src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
normalize.scss is usually used to make content look the same way in different browsers.